components/gitpod-db/src/typeorm/migration/1621920691254-InitialComSetup.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the Gitpod Enterprise Source Code License,
+ * See License.enterprise.txt in the project root folder.
+ */
+
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class InitialComSetup1595232295133 implements MigrationInterface {
+
+    public async up(queryRunner: QueryRunner): Promise<any> {
+        // TODO: FIXME remove these test queries
+        await queryRunner.query("CREATE TABLE IF NOT EXISTS `d_b_account_entry` (`id` int(11) DEFAULT NULL, `userId` char(36) NOT NULL, `amount` double NOT NULL, `date` varchar(255) NOT NULL, `expiryDate` varchar(255) NOT NULL DEFAULT '', `kind` char(7) NOT NULL, `description` text, `uid` char(36) NOT NULL, `creditId` char(36) DEFAULT NULL, `_lastModified` timestamp(6) NOT NULL DEFAULT CURRENT_TIMESTAMP(6) ON UPDATE CURRENT_TIMESTAMP(6), PRIMARY KEY (`uid`), KEY `ind_dbsync` (`_lastModified`), KEY `ind_expiryDate` (`expiryDate`)) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;");
```

Comment:
Is this taken from the live prod/staging DB? Or from source?

Reply:
from source. do you expect differences?